package.json

```diff
@@ -98,5 +98,8 @@
         "typescript": "^4.7.4",
         "ws": "^7.5.8"
     },
-    "packageManager": "npm@8.15.0"
+    "packageManager": "npm@8.15.0",
+    "dependencies": {
+        "@apify/storage-local": "^2.1.0"
+    }
```

Comment:
lets revert this, it gets added when you run e2e tests locally, but its not needed otherwise (and i dont want to install sqlite in the CI unless its the e2e workflow)

Reply:
Already did